src/Command/FilesCommand.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * @license
+ *
+ * Copyright (C) debricked AB
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code (usually found in the root of this application).
+ */
+
+namespace App\Command;
+
+use App\Service\FileGroupFinder;
+use Debricked\Shared\API\API;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Exception\DirectoryNotFoundException;
+use Symfony\Contracts\HttpClient\Exception\HttpExceptionInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+class FilesCommand extends Command
+{
+    use Style;
+
+    protected static $defaultName = 'debricked:files';
```

Comment:
Class name and command don't describe what the command does. Maybe to use `FindFilesCommand` and `debricked:files:find`?

Reply:
Good suggestion - I will change that